scripts/python/lib/argparse_gen.py

```diff
@@ -43,7 +43,7 @@ class Cmd(Enum):
     DEPLOY = DEPLOY_CMD
 
 
-def get_args(parser_args=False):
```

Comment:
Removing this optional argument precludes sphinx from running correctly.

Reply:
We seem to have gotten out of sync in the switch off of Gerrit.  I'll update this